app/util/remote_runner.tsx

```diff
@@ -53,6 +69,16 @@ export function triggerRemoteRun(invocationModel: InvocationModel, command: stri
     });
 }
 
+// commandWithRemoteRunnerFlags adds useful flags to bazel commands run on a remote runner
+export function commandWithRemoteRunnerFlags(command: string): string {
+  // The buildbuddy_ configs point to the corresponding env that created the workflow
+  // action (Ex. If it was created in dev, points to the dev app)
+  const addlFlags =
```

Comment:
can we document the connection between this and `ci_runner/main.go` both here and there? this is kind of a brittle connection right now.

Reply:
(fine to do separate from this pr)

Reply:
Good idea - done